aten/src/ATen/native/cpu/ReduceOpsKernel.cpp

```diff
@@ -148,6 +148,24 @@ static void or_kernel_impl(TensorIterator& iter) {
     /*ident=*/false);
 }
 
+static void min_values_kernel_impl(TensorIterator& iter) {
+  AT_DISPATCH_ALL_TYPES(iter.type(), "min_values", [&] {
+    binary_kernel_reduce_vec(
+      iter,
+      [=](scalar_t a, scalar_t b) -> scalar_t { return std::min(a, b); },
+      [=](Vec256<scalar_t> a, Vec256<scalar_t> b) { return minimum(a, b); });
+  });
+}
+
+static void max_values_kernel_impl(TensorIterator& iter) {
+  AT_DISPATCH_ALL_TYPES(iter.type(), "min_values", [&] {
+    binary_kernel_reduce_vec(
+      iter,
+      [=](scalar_t a, scalar_t b) -> scalar_t { return std::max(a, b); },
```

Comment:
I, personally, am more comfortable with explicit captures in case something changes. Same with all the other functionals. It can have significant performance implications if you don't know exactly that's happening.